mir-ci/mir_ci/robot_resources/screencopy.resource

```diff
@@ -8,93 +9,100 @@ Library             ../robot_libraries/WaylandHid.py
 *** Keywords ***
 Get Center Of ${region}
     [Documentation]    Get the center point of a region.
+    ...
+    ...    The return value is a tuple (x, y) of integers.
     ${point}=    Evaluate
-    ...    {"x": (${region}[left] + ${region}[right]) / 2, "y": (${region}[top] + ${region}[bottom]) / 2}
+    ...    (int((${region}[left] + ${region}[right]) / 2), int((${region}[top] + ${region}[bottom]) / 2))
     RETURN    ${point}
 
 Displace ${point} By (${x}, ${y})
-    [Documentation]    Shift a point by the specified displacements along the x and y axes.
-    ${point}=    Evaluate    {"x": ${point}[x] + ${x}, "y": ${point}[y] + ${y}}
+    [Documentation]    Shift a point by the specified displacements along
+    ...    the x and y axes.
+    ...
+    ...    ${point} must be a tuple (x, y).
+    ...
+    ...    The return value is the displaced point, represented as a tuple
+    ...    (x, y) of integers.
+    ${point}=    Evaluate    (int(${point}[0] + ${x}), int(${point}[1] + ${y}))
     RETURN    ${point}
 
-Move Pointer To ${destination}
-    [Documentation]    Move the pointer to a position or image template.
+Move Pointer To ${destination:^\(?!\(.+,.+\)\).+}
```

Comment:
It's not pretty and maybe there's a better of doing it: it matches "anything that is not (x, y)" to distinguish it from `Move Pointer To (${x}, ${y})`. Alternatively, we could parse the string in Python, but it would be more work.